tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1121,19 +1122,48 @@ public function theUserTriesToCreateASubFolderUsingTheGraphApi(
 	 *
 	 * @throws GuzzleException
 	 */
-	public function theUserHasCreateAFolderUsingTheGraphApi(
+	public function theUserHasCreatedAFolderInSpace(
 		string $user,
 		string $folder,
 		string $spaceName
 	): void {
-		$this->theUserCreatesAFolderUsingTheGraphApi($user, $folder, $spaceName);
-
+		$folder = \trim($folder, "/");
+		$paths = explode('/', $folder);
+		$folderPath = "";
+		foreach ($paths as $path) {
+			$folderPath .= "$path/";
+			$response = $this->createFolderInSpace($user, $folderPath, $spaceName);
+		}
```

Comment:
should we check every request for success? so the assertion must go inside the loop

Reply:
So, it's main purpose is to make sure that last folder is created